components/prospects/create-prospect-note.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react'
```

Comment:
The file name of a component should be in PascaCase, only the path to that file is in kabab-case. Always look at the existing files in the project